src/main/java/wiresegal/wob/arcanum/Parsing.kt

```diff
@@ -167,6 +174,60 @@ fun about(message: Message) {
     })
 }
 
+data class Progress(val title: String, val percentage: String, val link: String?)
+
+val progressCache = mutableListOf<Progress>()
+var progressCacheTimeStamp: Long? = null
+
+fun showProgressBar(message: Message) {
+    val progresses = extractProgresses()
+    val embed = EmbedBuilder ()
+            .setColor(embedColor)
+            .setTitle("Progress Bars")
+            .setUrl(homepageTarget)
+
+    val full = "█"
+    val empty = "░"
+    val sensitivity = 20
+
+    var progressInformation = ""
+    for ((name, percent, link) in progresses) {
+        val percentNumber = Integer.parseInt(percent)
+        val points = (percentNumber/100.0 * sensitivity).toInt()
+        val bar = full.repeat(points) + empty.repeat(sensitivity - points)
+        val title = if(link != null) "[$name]($link)" else name
+        progressInformation += "**$title**\n$bar   $percent%\n\n"
```

Comment:
Something that could be done here is padding the percentage to 3 characters using spaces, which would right-align it. It's not necessary, but would turn the polish to 11.

Reply:
This is unfortunately not possible since it's not a monospace font, so it won't line up properly.